### PR TITLE
[CUBRIDQA-33] refactor CTP and refine the parameter name 

### PR DESCRIPTION
**Pull request by supler**
@xkyu, @fanzaiqiang 

So far we have finished the CTP refactoring and CTP parameters enhancement, we would like to merge changes into master, please review it.

The change summary of CTP
1. added sql_by_cci, shell,ha_repl,isolation,jdbc and unittest components
2. refine the structure of readme
3. refine CTP parameter names 

The points of reviewing you may cover:
1. readme.md and all the configuration files

However, you can review codes or the structure of CTP if it's possible for your time. thanks

### CODE REVIEW

**Comment by xkyu** [comment]
@supler 
LGTM for the README and conf files.
Please merge the files after check below suggestions:
- Please add a space between '#' sign and comment for each line of the license stmt.
- There is no mention about JAVA_HOME in the prepare section for SQL test only.
It'd be better to use below stmt in prepare section of all tests.
'Make sure to set JAVA_HOME and required environment variables correctly'
-  There are many names for a test. (eg: 'sql', 'Sql', 'SQL', ...). Please unify the name of all tests in the README and use capital letters for name of the tests. ('SQL', 'MEDIUM', 'SHELL', etc ...)

File: CTP/sql/bin/run.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```
Comment (jinmeihua) [inline CTP/sql/bin/run.sh]: 1. Please add a space between '#' sign and comment for each line of the license stmt.
2. Is it still need the "set -x" ?

File: CTP/conf/ha_repl.conf [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#    derived from this software without specific prior written permission.
+#
+#THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+#INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+#DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+#SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+#SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
+#WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+#USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```
Comment (jinmeihua) [inline CTP/conf/ha_repl.conf]: Please add a space between '#' sign and comment for each line of the license stmt.

File: CTP/jdbc/bin/run.sh [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+#    derived from this software without specific prior written permission.
+#
+#THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+#INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+#DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+#SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+#SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
+#WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+#USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```
Comment (jinmeihua) [inline CTP/jdbc/bin/run.sh]: Please add a space between '#' sign and comment for each line of the license stmt.

File: CTP/common/src/com/navercorp/cubridqa/common/ConfigParameterConstants.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
+package com.navercorp.cubridqa.common;
+
```
Comment (jinmeihua) [inline CTP/common/src/com/navercorp/cubridqa/common/ConfigParameterConstants.java]: Please add the license stmt.

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/ShellHelper.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
+package com.navercorp.cubridqa.shell.main;
```
Comment (jinmeihua) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/ShellHelper.java]: Please add license stmt.

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java [hunk 6]
```diff
@@ -0,0 +1,279 @@
+package com.navercorp.cubridqa.shell.main;
```
Comment (jinmeihua) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/JdbcLocalTest.java]: Please add license stmt.

File: CTP/shell/src/com/navercorp/cubridqa/shell/main/GeneralLocalTest.java [hunk 7]
```diff
@@ -4,26 +4,39 @@
 import java.util.ArrayList;
```
Comment (jinmeihua) [inline CTP/shell/src/com/navercorp/cubridqa/shell/main/GeneralLocalTest.java]: Please add license stmt.

File: CTP/shell/src/com/navercorp/cubridqa/shell/common/GeneralFeedback.java [hunk 8]
```diff
@@ -0,0 +1,201 @@
+package com.navercorp.cubridqa.shell.common;
```
Comment (jinmeihua) [inline CTP/shell/src/com/navercorp/cubridqa/shell/common/GeneralFeedback.java]: Please add license stmt.

**Comment by jinmeihua** [comment]
@supler 

In addition to the above mentioned comments, the following files has no license statement too.
CTP/sql/memory/default.supp
CTP/shell/src/com/navercorp/cubridqa/shell/main/ShellHelper.java
CTP/shell/init_path/cubrid
CTP/shell/lib/MANIFEST.MF
CTP/shell/init_path/recovery_ignore_item.conf
CTP/common/script/upgrade.sh

It would be better to check the all files in the project. Since I just reviewed from the "file changes".
Thank you.

**Pull request merged**